addons/account/migrations/13.0.1.1/post-migration.py

```diff
@@ -1044,9 +1044,9 @@ def migrate(env, version):
     fill_res_partner_ranks(env)
     try_to_fill_account_account_tag_country_id(env)
     create_account_tax_repartition_lines(env)
-    move_tags_from_taxes_to_repartition_lines(env)
+    # move_tags_from_taxes_to_repartition_lines(env)
     assign_tax_repartition_line_to_move_lines(env)
-    assign_account_tags_to_move_lines(env)
+    # assign_account_tags_to_move_lines(env)
```

Comment:
2 dòng này có tác dụng map các thẻ tài khoản mà đang gắn trực tiếp với thuế sang các dòng trên thuế, sau đó map các thẻ với các phát sinh kế toán tương ứng.
Cách map này hiện tại không phù hợp với nghiệp vụ và các thẻ tài khoản đang có trên dữ liệu của khách hiện tại nên em comment lại đoạn này.
Người dùng có thể tự tạo ra các thẻ mới và gắn theo ý muốn, chưa rõ trong trường hợp đó thì cách map hiện tại của OCA có hợp lý không nên em chưa xóa 2 hàm này.